app/gui2/src/util/reactivity.ts

```diff
@@ -158,4 +159,5 @@ export function syncSet<T>(target: Set<T>, newState: Set<T>) {
   for (const newKey of newState) if (!target.has(newKey)) target.add(newKey)
 }
 
-export type ToValue<T> = Parameters<typeof toValue<T>>[0]
+/** Type of the parameter of `toValue`. */
+export type ToValue<T> = MaybeRefOrGetter<T> | ComputedRef<T>
```

Comment:
This should also be imported in `placement.ts`.

Reply:
#9924 removes all the redundant `ToValue` implementations, I just copied the new definition in `@/util/reactivity` to this PR so that I can use it while waiting for that to merge.